designer/client/data/index.ts

```diff
@@ -1,3 +1,8 @@
+/**
+ * This is a replacement for `data-model` which was a class used for mutating {@link FormDefinition} before it is saved.
```

Comment:
Hi. Suggest the below, if it is correct?

* This replaces the class `data-model` (deprecated) which we previously used to mutate {@link FormDefinition} before it is saved.